packages/terra-clinical-onset-picker/src/OnsetPicker.scss

```diff
@@ -1,4 +1,16 @@
+@import '~terra-breakpoints/lib/media-queries';
```

Comment:
let's add modules before the file extension. `OnsetPicker.module.scss`.

more discussion about that here: https://github.com/cerner/terra-core/issues/1511

Reply:
module extension added